File: websocket/src/main/java/com/github/websocket/payload/Specification.java
package com.github.websocket.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Specification {

    private Long id;

    private String name;

    private String description;

}

File: server-mocks/src/main/kotlin/com/github/server/mocks/payload/Specification.kt
package com.github.server.mocks.payload

class Specification(
        val id: Long,
        val name: String,
        val description: String
) {

    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (javaClass != other?.javaClass) return false

        other as Specification

        if (id != other.id) return false
        if (name != other.name) return false
        if (description != other.description) return false

        return true
    }

    override fun hashCode(): Int {
        var result = id.hashCode()
        result = 31 * result + name.hashCode()
        result = 31 * result + description.hashCode()
        return result
    }

    override fun toString(): String {
        return "Specification(id=$id, name='$name', description='$description')"
    }

}